api/python/stats.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Place Statistics
+nav_order: 10
+parent: Python
+grand_parent: API
+---
+
+# Get Statistics for Places.
+
+## `get_stats(dcids, stats_var, obs_dates='latest')`
+
+Returns a time series of statistical values for queried places based on the
+[`StatisticalVariable`](https://browser.datacommons.org/kg?dcid=StatisticalVariable) filtered by the specified dates.
```

Comment:
Do you want to add your documentation link about stats_var for better background?

Reply:
Yeah. Maybe future PR since that isn't checked in yet.